cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/io/cdap/cdap/etl/api/relational/StringExpression.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.cdap.etl.api.relational;
+
+import java.util.Objects;
+
+/**
+ * A default implementation of Expression that simply stores the expression as a string.
+ * The string is assumed to be ANSI SQL compliant.
+ * It is the responsibility of the factory creating StringExpression objects to enforce correctness of the expression.
+ */
+public class StringExpression implements Expression {
```

Comment:
This are implementations. It's better not to put them into API as this creates additional contract hard to refactor / extend.

Reply:
Do you suggest I should move this class inside the factory class instead?

Reply:
It should be in the google-cloud plugin, not in the cdap-etl-api

Reply:
Overall this is convenient implementation, but not part of API.

Reply:
So in this case `StringExpressionFactory` will have to move to google-cloud as well, correct?

Reply:
Removed, adding it in google-cloud now.